content/automate/ManageIQ/Cloud/VM/Retirement/Email.class/__methods__/vm_retire_extend.rb

```diff
@@ -15,6 +15,7 @@ def initialize(handle = $evm)
               end
 
               def main
+                @handle.log("warn", "[DEPRECATION] This method will be deprecated. Please use similarly named method from System/Notification/Email namespace.")
```

Comment:
@billfitzgerald0120 This is a class not a namespace. Since we are pointing to the email class we should say class.